src/handlers/payout/index.ts

```diff
@@ -38,8 +39,10 @@ export const handleIssueClosed = async () => {
   console.log({ assignee });
```

Comment:
it looks like you forget to remove console log lol

Reply:
I see haha